src/main/java/com/mulesoft/connector/github/internal/Converters/IssueConverter.java

```diff
@@ -2,16 +2,44 @@
 
 import com.google.gson.Gson;
 import com.google.gson.stream.JsonReader;
+import com.mulesoft.connector.github.api.Domain.IssueResponse;
 import com.mulesoft.connector.github.internal.Domain.Issue;
 
+import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.ArrayList;
 import java.util.List;
 
 public class IssueConverter {
-    public List<Issue> convertInputStreamToIssue (InputStream inputStream) throws IOException {
+
+    public IssueResponse convertInputStreamToIssue (InputStream inputStream) throws IOException {
+        if(inputStream != null){
```

Comment:
You can write an `if(inputStream == null) { return null; }` at the beginning so your code is more easy to read.